modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/package.json

```diff
@@ -32,39 +32,25 @@
 		"react-text-mask": "5.4.3",
 		"text-mask-addons": "3.8.0"
 	},
-	"devDependencies": {
```

Comment:
The only reason this wasn't failing before is probably because of SourceFormatter suppresions.

@albinojunior, could you please get rid of the exclusion so this `package.json` file follows everything else? Other possibly related errors will be `react` or `clay` version mismatches.

https://github.com/liferay/liferay-portal/blob/master/modules/dxp/apps/portal-workflow/source-formatter-suppressions.xml#L11

Reply:
Should I just remove the suppression of `package.json`, right?

Reply:
Yes! :)